src/components/select/react/Select.tsx

```diff
@@ -0,0 +1,344 @@
+import React, { ReactNode, useRef, useState } from 'react';
+
+import classNames from 'classnames';
+
+import { mdiAlertCircle, mdiCheckCircle, mdiClose, mdiCloseCircle, mdiMenuDown } from '@mdi/js';
+
+import { COMPONENT_PREFIX } from 'LumX/core/react/constants';
+
+import { Chip, Dropdown, Icon, Placement, Size, Theme } from 'LumX';
+
+import { IGenericProps, getRootClassName } from 'LumX/core/react/utils';
+import { handleBasicClasses } from 'LumX/core/utils';
+
+/////////////////////////////
+
+/**
+ * The authorized variants.
+ */
+const enum SelectVariant {
+    input = 'input',
+    chip = 'chip',
+}
+
+/**
+ * Defines the props of the component.
+ */
+interface ISelectProps extends IGenericProps {
+    /**
+     * The list of selected values.
+     */
+    selectedValues: string[];
+
+    /**
+     * Whether the select (input variant) is displayed with error style or not.
+     */
+    hasError?: boolean;
+
+    /**
+     * The helper to display within the popover (last position).
+     */
+    helper?: string;
+
+    /**
+     * Whether the select is disabled or not.
+     */
+    isDisabled?: boolean;
+
+    /**
+     * Whether the select is opened or not.
+     */
+    isOpen?: boolean;
+
+    /**
+     * Whether the select (input variant) is displayed with valid style or not.
+     */
+    isValid?: boolean;
+
+    /**
+     * The select label.
+     */
+    label?: string;
+
+    /**
+     * Wether to let user select multiple values in the choices list or not.
+     */
+    multiple?: boolean;
```

Comment:
`isMultiple` pour garder la cohérence du préffixe boolean is/has ??

Reply:
Done